### PR TITLE
Adding the DevDocs Website

### PR DESCRIPTION
**Pull request by DavidLambauer**
This is the initial PR for adding the DevDocs website.

### CODE REVIEW

**Review by sprankhub — COMMENTED** [review]
Has the license stuff been checked?

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "name": "laravel/laravel.com",
```
Comment (Vinai) [inline composer.json:2]: This probably should be updated to `mage-os/devdocs` or something like this?  
The same for the description :)

**Review by Vinai — CHANGES_REQUESTED** [review]

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,51 @@
+APP_NAME=Laravel
```
[the remark this answers is not in the document]
Comment (DavidLambauer) [reply .env.example:1]: Changed it, although this is just the example env. On Prod, I configured a proper name.

File: composer.json (hunk 1 above)
Comment (DavidLambauer) [reply composer.json:2]: Yeh, good catch

**Review by Vinai — APPROVED** [review]
LGTM

**Comment by Vinai** [comment]
@sprankhub Yes, it is MIT.

**Pull request merged**